modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -974,6 +975,24 @@ public void testCfgGap() throws InterruptedException {
         checkTableWithData(newNode, "t2");
     }
 
+    /**
+     * The test for updating cluster configuration with the default value.
+     * Check that new nodes will be able to synchronize the local cluster configuration.
+     */
+    @Test
+    public void updateClusterCfgWithDefaultValue() {
+        IgniteImpl ignite = startNode(0);
+
+        RocksDbStorageEngineConfiguration dbStorageEngineConfiguration = ignite.clusterConfiguration()
+                .getConfiguration(RocksDbStorageEngineConfiguration.KEY);
+        int defaultValue = dbStorageEngineConfiguration.flushDelayMillis().value();
+        dbStorageEngineConfiguration.flushDelayMillis().update(defaultValue).join();
```

Comment:
Can you please add timeout to the join?
Another question - why have you decided to write an integration test instead of the unit test?